yt/chyt/controller/internal/agent/agent.go

```diff
@@ -385,10 +388,27 @@ func (a *Agent) background() {
 			a.healthState.SetTrackOpsState(nil)
 		case <-ticker.C:
 			a.pass()
+		case a.opletBatchCh <- a.getOpletMappingCopy():
+			continue
+		case scT := <-a.scalingTargetCh:
+			oplet, ok := a.aliasToOp[scT.OpletAlias]
+			if !ok {
+				a.l.Warnf("Oplet %s not found, skipping scaling")
```

Comment:
We usually don't use Warn**f**/Info**f**/etc. Our log is structured, so it's better to write
```
a.l.Warn("Oplet not found, skipping scaling", log.String("alias", scT.OpletAlias))
```